fdc.js

```diff
@@ -174,14 +176,14 @@ export function BaseDisc(fdc, name, data, flusher) {
         if (this.flusher) this.flusher();
     };
     BaseDisc.prototype.seek = (track) => {
-        this.seekOffset = track * 10 * 256;
+        this.seekOffset = track * this.sectorsPerTrack * 256;
         if (this.isDsd) this.seekOffset <<= 1;
         const oldTrack = this.track;
         this.track = track;
         return this.track - oldTrack;
     };
-    BaseDisc.prototype.check = (track, side, density) => {
-        if (this.track !== track || density || (side && !this.isDsd)) {
+    BaseDisc.prototype.check = (track, side) => {
```

Comment:
unless I'm mistaken are there calls to `check()` somewhere passing `density` ? I can't see that you removed the `density` param, though js is forgiving I'd prefer to remove that if possible.

Reply:
Hi there, yes the `check()` seemed to be using `density` just in order to create an error on non single-density disks. I've tidied up the various calls to it now it isn't needed to be passed in.

Reply:
awesome thanks!